ignite/metrics/metric.py

```diff
@@ -346,10 +435,16 @@ def _check_usage(self, usage: Union[str, MetricUsage]) -> MetricUsage:
         if isinstance(usage, str):
             if usage == EpochWise.usage_name:
                 usage = EpochWise()
+            elif usage == RunningEpochWise.usage_name:
```

Comment:
Let's make this code more generic, let's register all usages in a private list and here we can just iterate over this list.